backend/startup.py

```diff
@@ -43,43 +43,48 @@ def sentry_before_send(event, hint):
     )
 
 
-flask_app = create_flask_app()
-
+# I'm importing it here to make sure that the flask initializes before the fastapi one
+from v2.main import build_app  # noqa
```

Comment:
nit: I've historically done imports runtime imports in the scope of the function that needs them. It also makes it clearer why the import is not at the type of the module.